x/delegation/client/cli/query.go

```diff
@@ -277,3 +279,40 @@ func QueryAssociatedStakersByOperator() *cobra.Command {
 	flags.AddQueryFlagsToCmd(cmd)
 	return cmd
 }
+
+func QueryDelegatedStakersByOperator() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "QueryDelegatedStakersByOperator <operatorAddr> <assetID>",
+		Short: "Get the delegated stakers for the specified operator and asset",
+		Long:  "Get the delegated stakers for the specified operator and asset",
+		Args:  cobra.ExactArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientQueryContext(cmd)
+			if err != nil {
+				return err
+			}
+			accAddr, err := sdk.AccAddressFromBech32(args[0])
+			if err != nil {
+				return errorsmod.Wrap(types.ErrInvalidCliCmdArg, err.Error())
+			}
+			assetID := strings.ToLower(args[1])
+			if _, _, err := types.ValidateID(assetID, false, false); err != nil {
+				return errorsmod.Wrap(types.ErrInvalidCliCmdArg, err.Error())
+			}
+			queryClient := delegationtype.NewQueryClient(clientCtx)
+			req := &delegationtype.QueryDelegatedStakersByOperatorReq{
+				Operator: accAddr.String(),
+				AssetId:  assetID,
+			}
+			res, err := queryClient.QueryDelegatedStakersByOperator(context.Background(), req)
+			if err != nil {
+				return err
+			}
+			fmt.Println("the number of stakers is:", len(res.Stakers))
```

Comment:
Remove the debug

Reply:
It’s not debug information; it’s used to display the number of stakers when running this command

Reply:
It'll cause trouble when using `--output json`. Maybe include it in the result instead.

Reply:
Thanks, it will be fixed in the next commit.